### PR TITLE
f-media-element@0.1.0 - generated output for media element component

### PR DESCRIPTION
**Pull request by matthewhardern**
This is just the output from the new generator, start of a new component for some work on the new stamp cards system.

### CODE REVIEW

File: packages/components/molecules/f-media-element/src/components/MediaElement.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        :class="$style['c-mediaElement']"
+        data-test-id="mediaElement-component">
+        mediaElement
+    </div>
+</template>
+
+<script>
+
+export default {
+    name: 'MediaElement',
```
Comment (DamianMullins) [inline packages/components/molecules/f-media-element/src/components/MediaElement.vue]: Don't need to set name inside components, vue-cli adds this automatically.

File: packages/components/molecules/f-media-element/src/components/MediaElement.vue [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    components: {},
```
Comment (DamianMullins) [inline packages/components/molecules/f-media-element/src/components/MediaElement.vue]: I'd remove these properties if they aren't being used yet.

File: packages/components/molecules/f-media-element/src/components/MediaElement.vue (hunk 1 above)
Comment (matthewhardern) [reply packages/components/molecules/f-media-element/src/components/MediaElement.vue]: This is generated output, could you please add this to the generator PR as a comment thanks, will make the change :)

File: packages/components/molecules/f-media-element/src/components/MediaElement.vue (hunk 2 above)
Comment (matthewhardern) [reply packages/components/molecules/f-media-element/src/components/MediaElement.vue]: Should this also be a change in the generator? Will make change

File: packages/components/molecules/f-media-element/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+
+v0.1.0
+------------------------------
+*January 29, 2021*
+
+### Added
+- Add your change under the following headings: `Added`/`Changed`/`Deprecated`/`Removed`/`Fixed`/`Security`
```
Comment (FayeCarter) [inline packages/components/molecules/f-media-element/CHANGELOG.md]: Do you need to change this?

Comment (matthewhardern) [reply packages/components/molecules/f-media-element/CHANGELOG.md]: yes sorry will change :)

**Review by FayeCarter — APPROVED** [review]

**Review by dandel10n — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**